src/components/InfoFootnote.tsx

```diff
@@ -0,0 +1,25 @@
+import classNames from 'classnames';
+import React from 'react';
+
+type InfoFootnoteProps = {
+    className?: string;
+    isMobile?: boolean;
+    text: string;
+};
+
+const InfoFootnote = ({ className, isMobile, text }: InfoFootnoteProps) => (
+    <div
+        className={classNames('info-footnote', className, {
+            [`${className}--mobile`]: isMobile,
```

Comment:
Just asking: can't we use `@include mobile` in `.css` file instead of passing `isMobile` and adding `className`?